packages/svelte/src/compiler/errors.js

```diff
@@ -170,12 +170,8 @@ const runes = {
 	'invalid-legacy-export': () => `Cannot use \`export let\` in runes mode — use $props instead`,
 	/** @param {string} rune */
 	'invalid-rune-usage': (rune) => `Cannot use ${rune} rune in non-runes mode`,
-	'invalid-state-export': () =>
-		`Cannot export state if it is reassigned. Either export a function returning the state value or only mutate the state value's properties`,
-	'invalid-derived-export': () =>
-		`Cannot export derived state. To expose the current derived value, export a function returning its value`,
-	'invalid-prop-export': () =>
-		`Cannot export properties. To expose the current value of a property, export a function returning its value`,
+	'invalid-state-export': () => `Cannot export state from a module if it is reassigned`,
+	'invalid-derived-export': () => `Cannot export derived state from a module`,
```

Comment:
Code looks good now, though I slightly prefered the changed error messages because they gave you instructions on how to resolve the error. Thoughts?

Reply:
merged the changes from both